test/qpy_compat/test_qpy.py

```diff
@@ -242,6 +243,38 @@ def generate_single_clbit_condition_teleportation():  # pylint: disable=invalid-
     return teleport_qc
 
 
+def generate_parameter_vector():
+    """Generate tests for parameter vector element ordering."""
+    qc = QuantumCircuit(11)
+    input_params = ParameterVector("x_par", 11)
+    user_params = ParameterVector("θ_par", 11)
+    for i, param in enumerate(user_params):
+        qc.ry(param, i)
+    for i, param in enumerate(input_params):
+        qc.rz(param, i)
+    return qc
+
+
+def generate_parameter_vector_expression():  # pylint: disable=invalid-name
```

Comment:
Perhaps we can disable the upper limit on function name length?  It's frustrated me a couple of times too.